### PR TITLE
Copy changes

### LINKED ISSUE #245: Converting to date without time
I need to convert an ecto date, which looks like `2017-03-09 13:48:09.875620` to just the year, month day `2017-03-09`. 

Locally, this works fine with either `Date.to_string(@date_goes_here)`(first test below) or `Date.to_iso8601(@date_goes_here)`(second test below). However on travis this fails with similar errors:
image

There is something that's different about my local environment from travis - checking it out now.

Couple of things I noticed right away - travis.yml had 1.3.4 and I am running 1.4.0
mix.exs has `elixir: "~1.2"`

### PR DESCRIPTION
**Pull request by katbow**
Update goals and coping strategies text formatting.

#209 #210 #212 #237

fixes #245

### CODE REVIEW

**Comment by katbow** [comment]
oh man. tests failing on travis but not locally 😢

File: web/templates/goal/show.html.eex [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <div class="ma3 ma4-m ma4-l mv2 mv3-m mv4-l">
     <b class="di"><%= markdown String.trim_trailing(@goal.content, " #Goal") %></b>
     <%= link "Edit", to: goal_path(@conn, :edit, @goal), class: "di" %>
-    <p class="mv3">Created <%= @goal.inserted_at %></p>
+    <p class="mv3">Created <%= String.slice(Date.to_string(@goal.inserted_at), 0..10) %></p>
```
Comment (tonydaly) [inline web/templates/goal/show.html.eex]: @katbow I've got a lil helper for this in the PostView, so you can do `time_ago_in_words(@goal.inserted_at)`.

Comment (katbow) [inline web/templates/goal/show.html.eex]: I think in this case they want it as the actual date. I will move it to the view thought all the same 👍

Comment (tonydaly) [inline web/templates/goal/show.html.eex]: Ahh ok, I'd still take a look at the [Timex](https://github.com/bitwalker/timex) library which I'm already using as it allows for flexible date formatting.

**Comment by katbow** [comment]
@tonydaly it looks like it was because of a diff in elixir version from what I was running vs what was in travis.yml. Is it okay to update that, or should I keep that the same? If so, should I just try reformatting the date with Timex instead?

**Comment by tonydaly** [comment]
Yeah @nelsonic and I discussed it before, the version of Elixir that we're using is defined in a few places and we should really standardise it, not sure the best approach to this tho. We should probably try to keep up to date with the latest version, so keep that travis change.

But I'd still try using Timex and it'll give you greater control over the format using the pretty standard `strftime`.

**Comment by katbow** [comment]
okay thanks! I'll check it out now.

**Comment by katbow** [comment]
I think I'm going to pass on Timex now - I'll keep it in mind if I start doing more intense operations with dates/times. I think the problem is solved right now with standard functions.

File: web/views/goal_view.ex [hunk 2]
```diff
@@ -1,9 +1,14 @@
 defmodule Healthlocker.GoalView do
   use Healthlocker.Web, :view
+  use Timex
```
Comment (tonydaly) [inline web/views/goal_view.ex]: No worries, no point using it if we dont need to. But I'd just remove this unused import.

Comment (katbow) [inline web/views/goal_view.ex]: done 🙂

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 language: elixir
 elixir:
-- 1.3.4
+- 1.4.0
```
Comment (nelsonic) [inline .travis.yml]: 👍

File: web/views/goal_view.ex [hunk 4]
```diff
@@ -6,4 +6,8 @@ defmodule Healthlocker.GoalView do
     |> Earmark.as_html!
     |> raw
   end
+
+  def pretty_date(date) do
+    Date.to_iso8601(date)
```
Comment (nelsonic) [inline web/views/goal_view.ex]: in-line comment here would be _really_ good to help the next person know why this date format was selected... 😉

**Pull request merged**